docs/merkletree/treetex/main.go

```diff
@@ -185,15 +204,20 @@ func drawLeaf(prefix string, index uint64, leafText, dataText nodeTextFunc) {
 	id := compact.NewNodeID(0, index)
 	a := nInfo[id]
 
-	// First render the leaf node of the Merkle tree
-	a.rangeIndices = nil
-	a.incPath = false
+	// First render the leaf node of the Merkle tree.
+	if len(a.dataRangeIndices) > 0 {
+		a.incPath = false
+	}
+	if !a.proof {
+		a.dataRangeIndices = nil
+	}
```

Comment:
I don't follow the connection between ranges and proofs here. Maybe comment a bit?

Reply:
I think this was left-over from when there was only the one range, I've dinged it.